src/components/Categories.js

```diff
@@ -1,5 +1,14 @@
-const Categories = () => (
-  <button type="button" className="check-status">Check Status</button>
-);
+import { useSelector } from 'react-redux';
+
+const Categories = () => {
+  const status = useSelector((state) => state.category.status);
+  return (
+    <>
+      <h1>{status}</h1>
+      <button type="button" className="check-status">Check Status</button>
```

Comment:
- [ ] This button on the categories page should dispatch an action when clicked. Right now the "Under construction" string is readily displayed when I visit the page. But that should not be so. The text should not be visible until I dispatch an action to the store which displays the text. Kindly refactor for that.

Reply:
@TedLivist Hi Ted, thank you for the feedback. I will implement the review changes.